### PR TITLE
Added a Tutorial

### PR DESCRIPTION
**Pull request by xtellurian**
Contains files under `docs/tutorial` plus minor edits to the main README file.

tutorial breaks down how to use the demo pod contained in deploy/demo

The three major stages are:
- Creating & configuring an Azure Kubernetes Service using the CLI
- Configuring AKS with the right k8s infra and crd
- Creating an MSI with Azure CLI then using it in AKS with the demo pod

### CODE REVIEW

File: docs/tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+$ kubectl logs mic-64ddcf5f65-h4hft
+....
+I0606 23:19:45.867711       1 crd.go:123] Got id podid to assign
+I0606 23:19:45.867829       1 crd.go:142] Creating assigned Id: demo-5788d95785-ghzwv-default-podid
+I0606 23:19:45.874002       1 cloudprovider.go:170] Find aks-nodepool1-15831963-0 in resource group: MC_k8s-test_clusterFrank_eastus
+I0606 23:20:11.051552       1 cloudprovider.go:162] Underlying cloud provider operation took 25.04421296s
+I0606 23:20:11.051846       1 mic.go:259] Sync took: 25.220821436s
+I0606 23:20:11.052905       1 event.go:218] Event(v1.ObjectReference{Kind:"AzureIdentityBinding", Namespace:"default", Name:"myIdBinding", UID:"19a07e0e-69e0-11e8-9e9f-4addade2df92", APIVersion:"aadpodidentity.k8s.io/v1", ResourceVersion:"89529", FieldPath:""}): type: 'Normal' reason: 'binding applied' Binding myIdBinding applied on node aks-nodepool1-15831963-0 for pod demo-5788d95785-ghzwv-default-podid
+
+```
```
Comment (kkmsft) [inline docs/tutorial/README.md]: We can see the same events here as well, either:

kubectl get events --field-selector=involvedObject.kind=AzureIdentityBinding

or 

kubectl describe azureidentitybinding

File: docs/tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+### 2.1. Install Custom Resource Definitions
+
+The following script installs three [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) on our kubernetes cluster.
+
+`./scripts/2-config-aks/1-install-crd.sh`
+
+The three CRDs are:
+
+- azureAssignedIdentity: Do not create these, they will be auto-created by the azureIdentityController
```
[the remark this answers is not in the document]
Comment (xtellurian) [reply docs/tutorial/README.md]: added a comment to that affect

File: docs/tutorial/README.md (hunk 1 above)
Comment (xtellurian) [reply docs/tutorial/README.md]: `kubectl get events --field-selector=involvedObject.kind=AzureIdentityBinding` didn't work for me

but I added docs saying to try the describe command.

**Comment by kkmsft** [comment]
Thank you, very much for this PR -@xtellurian. Have merged it 😀

**Pull request merged**